package.json

```diff
@@ -21,7 +21,7 @@
     "tslint-config-prettier": "^1.18.0",
     "tslint-consistent-codestyle": "^1.15.1",
     "tslint-plugin-prettier": "^2.0.1",
-    "typescript": "^3.4.5"
+    "typescript": "~3.4.5"
```

Comment:
Had to do this because upgrading to `3.5.x` broke the build, and may introduce breaking changes for Buidler users.